src/components/SetStateDialog.vue

```diff
@@ -52,103 +52,102 @@ export default {
+              style="margin-bottom: -30px;"
+            ></v-checkbox>
+            <v-checkbox
+              v-if="flowRun"
+              v-model="childTasks"
+              :disabled="!allTasks"
+              label="Include mapped task runs - this may take some time"
+            ></v-checkbox>
+          </div>
+
+          <v-card-actions>
+            <v-spacer></v-spacer>
+            <v-tooltip top>
+              <template #activator="{ on, attrs }">
+                <v-btn
+                  v-disable-read-only-user="!selectedState"
+                  :loading="markAsLoading"
+                  color="primary"
+                  v-bind="attrs"
+                  v-on="on"
+                  @click="changeState"
+                >
+                  Confirm
+                </v-btn>
+              </template>
+
+              <span v-if="childTasks" class="pt-1">
+                If you have a lot of child mapped tasks, the state change may
+                continue in the background.
+                <br />
+              </span>
+
+              <span v-if="dialogType === 'task run'">
+                This may have an effect on downstream tasks.
+                <br />
+              </span>
 
-              <v-stepper-content step="2">
-                <v-card-text>
-                  Please be aware that clicking on confirm will set the state of
-                  your
-                  {{ dialogType }}
-                  {{ taskRun ? taskRun.name : flowRun.name }} to
-                  <span class="font-weight-black pb-8">
-                    {{ selectedState }}.</span
-                  >
-                  <div v-if="childTasks" class="pt-1">
-                    If you have a lot of child mapped tasks, the state change
-                    may continue in the background.</div
-                  >
-                  <span v-if="dialogType === 'task run'">
-                    This may have an effect on downstream tasks.
-                  </span>
-                </v-card-text>
-                <v-card-actions>
-                  <v-spacer />
-                  <v-btn
-                    v-disable-read-only-user="!selectedState"
-                    color="primary"
-                    @click="changeState"
-                  >
-                    Confirm
-                  </v-btn>
-                  <v-btn text @click="reset">Cancel</v-btn>
-                </v-card-actions>
-              </v-stepper-content>
-            </v-stepper-items>
-          </v-stepper>
-        </v-card-text>
+              Please be aware that clicking on confirm will set the state of
+              your
+              {{ dialogType }}
+              {{
+                taskRun && taskRun.name
+                  ? taskRun.name
+                  : taskRun
+                  ? taskRun.task.name
+                  : flowRun.name
+              }}
+              to
+              <span class="font-weight-black pb-8"> {{ selectedState }}.</span>
```

Comment:
I actually think we could take all of this out.  I know it was in the old version and the tooltip is a great addition for the  necessary warnings about child and downstream tasks but I'm not sure this section/warning adds much and possibly detracts from the more important warnings.